twpca/crossval.py

```diff
@@ -53,29 +52,30 @@ def cross_validate(model, data, nfits, drop_prob, **fit_kw):
         # fit the model to the training set
         tf.reset_default_graph()
         sess = tf.Session()
-        model.fit(traindata, sess=sess, **fit_kw)
+        model = TWPCA(traindata, **model_kw)
+        model.fit(**fit_kw)
 
         # compute mean error on test set
         resid = data - model.predict()
         test_error = np.mean(resid[testmask]**2)
 
         results.append({
              'params': model.params,
-             'train_error': model._sess.run(model.recon_cost),
+             'train_error': model.recon_cost,
              'test_error': test_error,
              'obj_history': model.obj_history
             }
         )
 
         # clean-up tensorflow
-        sess.close()
+        model._sess.close()
 
     return results
 
 def hyperparam_search(data, n_components, warp_scales, time_scales,
                       nfits=1, drop_prob=0.1, warp_reg=None, time_reg=None,
                       fit_kw=dict(lr=(1e-1, 1e-2), niter=(250, 500), progressbar=False),
-                      **model_kw):
+                      model_kw=dict()):
```

Comment:
don't have a mutable as a kwarg, set it to None and then assign it dict() if it is None